assembly/allowed-licenses.json

```diff
@@ -0,0 +1,32 @@
+{
+  "allowedLicenses": [
+    {"moduleLicense": "Apache License, Version 2.0"},
+    {"moduleLicense": "Apache Software License, version 1.1"},
+    {"moduleLicense": "The MIT License"},
+    {"moduleLicense": "The 3-Clause BSD License"},
+    {"moduleLicense": "The 2-Clause BSD License"},
+
+    {"moduleLicense": "BSD New license"},
+    {"moduleLicense": "BSD"},
+    {"moduleLicense": "New BSD License"},
+    {"moduleLicense": "BSD New license"},
```

Comment:
It seems to be duplicated with the [line 9](https://github.com/h2oai/sparkling-water/pull/1256/files#diff-e71bc2016e24993261af924f4072d31bR9).